be/src/vec/columns/predicate_column.h

```diff
@@ -205,13 +218,30 @@ class PredicateColumnType final : public COWHelper<IColumn, PredicateColumnType<
         }
     }
 
+    void insert_many_date(const char* data_ptr, size_t num) {
+        size_t intput_type_size  = sizeof(uint24_t);
+        size_t res_type_size = sizeof(uint32_t);
+        char* input_data_ptr = const_cast<char*>(data_ptr);
+        
+        char* res_ptr = (char*)data.get_end_ptr();
+        memset(res_ptr, 0, res_type_size * num);
+        for (int i = 0; i < num; i++) {
```

Comment:
Why not batch memcpy?

Reply:
```input_data_ptr``` is a uint24_t array.
```res_ptr``` is a uint32_t array.
The element length is different.

Reply:
ok